client/gutenberg/editor/utils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { requestHttpData } from 'state/data-layer/http-data';
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+/**
+ * Request a site's frame nonce from the v1.3 endpoint.
+ *
+ * @param {number} siteId  Site Id.
+ * @return {*} Stored data container for request.
+ */
+export const getFrameNonce = siteId =>
+	requestHttpData(
+		`frame-nonce-${ siteId }`,
+		http( {
+			type: WPCOM_HTTP_REQUEST,
```

Comment:
No need to override the type property here on this action, see https://github.com/Automattic/wp-calypso/blob/63f5373c514d3f9bd578f854bb199cbc9bbdc81c/client/state/data-layer/wpcom-http/actions.js#L60